analyses/collapse-rnaseq/cwl/scripts/convert_to_rds.R

```diff
@@ -0,0 +1,32 @@
+if(!require(optparse)){install.packages('optparse')}
```

Comment:
Would this work in a cwl runtime?

Reply:
Yes, in the example workflow, even though I have modified it since then, it works fine. From log:
```
2023-03-14T17:19:43.544739160Z: Loading required package: optparse 2023-03-14T17:19:43.547572677Z: Installing package into ‘/usr/local/lib/R/site-library’ 2023-03-14T17:19:43.547588528Z: (as ‘lib’ is unspecified) 2023-03-14T17:19:44.898506173Z: also installing the dependency ‘getopt’ 
```

Reply:
This seems wild to me. I always imagine the cwl runtimes as locked down but I guess that's not really the case.

Reply:
Like in terms of installing stuff at run time? We used to do that with the VEP docker. If you block network access in your project, then yeah, this is a problem